code/datums/components/overlay_lighting.dm

```diff
@@ -135,6 +136,7 @@
 		COMSIG_ATOM_SET_LIGHT_ON,
 		COMSIG_ATOM_SET_LIGHT_FLAGS,
 		COMSIG_ATOM_USED_IN_CRAFT,
+		COMSIG_LIGHT_EATER_ACT,
```

Comment:
Are you are registering to `COMSIG_LIGHT_EATER_QUEUE` and unregistering to `COMSIG_LIGHT_EATER_ACT`
This does not seem intended

Reply:
Fixed.